synapse/cortex.py

```diff
@@ -1493,14 +1475,58 @@ async def initServicePassive(self):
         for layer in self.layers.values():
             await layer.initLayerPassive()
 
+        await self.finiStormPool()
+
+    async def initStormPool(self):
+
+        byts = self.slab.get(b'storm:pool', db='cell:conf')
+        if byts is None:
+            return
+
+        url, opts = s_msgpack.un(byts)
+
+        self.stormpoolurl = url
+        self.stormpoolopts = opts
+
+        self.stormpool = await s_telepath.open(url)
+
+        # make this one a fini weakref vs the fini() handler
+        self.onfini(self.stormpool)
+
+    async def finiStormPool(self):
+
         if self.stormpool is not None:
             await self.stormpool.fini()
             self.stormpool = None
 
-    async def initQueryMirror(self):
-        if self.stormpoolurl is not None:
-            self.stormpool = await s_telepath.open(self.stormpoolurl)
-            self.onfini(self.stormpool.fini)
+    async def setStormPool(self, url, opts):
+        return await self._push('set:storm:pool', url, opts)
+
+    async def getStormPool(self):
+        byts = self.slab.get(b'storm:pool', db='cell:conf')
+        if byts is None:
+            return None
+        return s_msgpack.un(byts)
+
+    @s_nexus.Pusher.onPushAuto('storm:pool:set')
+    async def setStormPool(self, url, opts):
```

Comment:
```suggestion
    @s_nexus.Pusher.onPush('storm:pool:set')
    async def _setStormPool(self, url, opts):
```
Seems like this is supposed to be the bottom side nexus handler? The same function is defined above on L1502.

Reply:
the "top half" function was vestigial. I removed it, thanks!